lib/Jsrt/Core/JsrtContextCore.cpp

```diff
@@ -176,6 +176,23 @@ HRESULT ChakraCoreHostScriptContext::InitializeImportMeta(Js::ModuleRecordBase*
     return E_INVALIDARG;
 }
 
+bool ChakraCoreHostScriptContext::ReportModuleCompletion(Js::ModuleRecordBase* module, Js::Var exception)
+{
+    if (reportModuleCompletionCallback == nullptr)
+    {
+        return false;
+    }
```

Comment:
If the host has not set the callback just return false - this is to avoid backwards compatibility breaks.